### ISSUE TITLE
Display italicised text in headlines & subheadings

### ISSUE THREAD

**Issue by bcmn** [bbc/simorgh#1276]
**Is your feature request related to a problem? Please describe.**
Blocked on and https://github.com/bbc/simorgh/issues/1356 May also require further work on Psammead-headings -- i.e. a CSS rule added so child `<i>` tags use the correct font-family? 
Headlines & subheadlines should respect the "emphasis" text fragment attribute.

**Describe the solution you'd like**
When an article has a headline or subheading containing text marked with the "emphasis", we will display the appropriate Reith italics font, with the string contained inside an `<i>` tag, for semantic information.

Currently we will ignore any other rich-text option that appear in headlines or subheadings.

If not handled by #1356, we will need to be able to conditionally render certain fragment attributes.

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

James suggested at story refinement that we will want to do a quick regression across different devices & browsers, ensuring it’s loaded correctly for font loading.

From Catharine: Consider accessibility tests, ensuring we understand the trade-off between the semantic information offered by `<i>` tags, & the "text splitting" behaviour.

Dev insight: Feeling that unit tests might be sufficient -- though worth discussing if we want to test html structure for presence of `i` and `b` tags and following behaviour (do we have test data with this scenario?).

**Additional context**

- [x] Initially labelled with ["Refinement needed"](https://github.com/bbc/simorgh/labels/Refinement%20Needed)

**Comment by sareh** [comment]
Blocked on https://github.com/bbc/psammead/issues/281

**Comment by pjlee11** [comment]
While in a refinement session looking at this issue, myself and @sareh realised that the Headings container currently doesn't render fragments. Because of this we have a [new issue](https://github.com/bbc/simorgh/issues/1356) which will block this one

**Comment by bcmn** [comment]
No longer blocked on [Psammead#285](https://github.com/bbc/psammead/issues/285).
